API/Services/Implementations/GamesService.cs

```diff
@@ -0,0 +1,32 @@
+using API.DTOs;
+using API.Services.Interfaces;
+using AutoMapper;
+using Database.Entities;
+using Database.Repositories.Interfaces;
+
+namespace API.Services.Implementations;
+
+public class GamesService(IGamesRepository gamesRepository, IMapper mapper) : IGamesService
+{
+    public async Task<GameDTO?> GetAsync(int id) =>
+        mapper.Map<GameDTO?>(await gamesRepository.GetAsync(id));
+
+    public async Task<GameDTO?> UpdateAsync(int id, GameDTO game)
+    {
+        Game? existing = await gamesRepository.GetAsync(id);
+        if (existing is null)
+        {
+            return null;
+        }
+
+        existing.Ruleset = game.Ruleset;
+        existing.ScoringType = game.ScoringType;
+        existing.TeamType = game.TeamType;
+        existing.Mods = game.Mods;
+        existing.StartTime = game.StartTime;
+        existing.EndTime = game.EndTime ?? existing.EndTime;
```

Comment:
Same here as with `GameScoresService.UpdateAsync`, should be able to update processing enums